### ISSUE TITLE
Revive `test_Far_GetConfig()` in macrotest.lua.

### ISSUE THREAD

**Issue by MKadaner** [FarGroup/FarManager#624]
### Far Manager version

3.0.6099.0

### OS version

N/A

### Steps to reproduce

Uncomment [the test](https://github.com/FarGroup/FarManager/blob/833c93989cc63b3fde1a9f85552bd4d730db2b7b/plugins/luamacro/macrotest.lua#L1061) and run ci pipeline.

### Expected behavior

The test passes.

### Actual behavior

The test fails.

**Comment by alabuzhev** [comment]
@shmuz would you happen to remember why it was commented in 6b808a110570692552a1a722ae07f5a94124a518?

**Comment by shmuz** [comment]
AFAICR I did it because it became a boring task to regularly follow after the frequent changes (deletions, additions, renames) in the configuration variables. Yes I accepted that those changes (as opposed to API changes) were legal but it was too boring.

**Comment by MKadaner** [comment]
I respectfully disagree. The test as it is, is a dead code. It should either be exhumated or exterminated.
